setup.py

```diff
@@ -8,7 +8,16 @@
 from packaging.version import parse, Version
 import setuptools
 import torch
-from torch.utils.cpp_extension import BuildExtension, CUDAExtension, CUDA_HOME, ROCM_HOME
+import sys
+sys.path.append("./")
+BUILD_CPU_ONLY = os.getenv('VLLM_BUILD_CPU_ONLY', "0") == "1"
+BUILD_XPU_OPS = os.getenv('VLLM_BUILD_XPU_OPS', "0") == "1"
+if BUILD_XPU_OPS:
+    from xpu_extension.xpu_cpp_extension import DPCPPExtension, DpcppBuildExtension 
```

Comment:
Can utilize extensions what ipex already provides: 
https://intel.github.io/intel-extension-for-pytorch/xpu/latest/tutorials/features/DPC%2B%2B_Extension.html

Reply:
thanks for your comments. already updated. please check latest change.